src/main/scala/beam/replanning/ReplanningUtil.scala

```diff
@@ -103,7 +103,7 @@ object ReplanningUtil {
     for (i <- 0 until originalPlan.getPlanElements.size() - 1) {
       newPlan.getPlanElements.add(originalPlan.getPlanElements.get(i))
       if (tripsLength > i) {
-        val newLeg = PopulationUtils.createLeg(trips(i).tripClassifier.matsimMode)
+        val newLeg = PopulationUtils.createLeg(trips(i).tripClassifier.value)
```

Comment:
According to the following code: https://github.com/LBNL-UCB-STI/beam/blob/develop/src/main/scala/beam/agentsim/agents/household/HouseholdActor.scala#L245 we create a non-matsim mode. So yes, it's good to go